app/homepage/ui.test.js

```diff
@@ -65,3 +65,107 @@ test('should navigate to the browse solution page when clicking on the browse pr
+  await t
+    .expect(buyersGuideLink.exists).ok()
+    .click(buyersGuideLink)
+    .expect(getLocation()).contains('/guide');
+});
+
+test('should navigate guide page contact us section', async (t) => {
+  await pageSetup(t);
+  const getLocation = ClientFunction(() => document.location.href);
+  const guideContactUsLink = Selector('[data-test-id="footer-component"] li:nth-child(2) > a');
+  await t
+    .expect(guideContactUsLink.exists).ok()
+    .click(guideContactUsLink)
+    .expect(getLocation()).contains('/guide#contact-us');
+});
+
+test('should navigate nhs digital page', async (t) => {
+  await pageSetup(t);
+  const getLocation = ClientFunction(() => document.location.href);
+  const nhsDigitalLink = Selector('[data-test-id="footer-component"] li:nth-child(3) > a');
+  await t
+    .expect(nhsDigitalLink.exists).ok()
+    .click(nhsDigitalLink)
+    .expect(getLocation()).contains('https://digital.nhs.uk/');
+});
+
+test('should navigate to about GPIT futures page', async (t) => {
+  await pageSetup(t);
+  const getLocation = ClientFunction(() => document.location.href);
+  const aboutGpitLink = Selector('[data-test-id="footer-component"] li:nth-child(4) > a');
+  await t
+    .expect(aboutGpitLink.exists).ok()
+    .click(aboutGpitLink)
+    .expect(getLocation()).contains('https://digital.nhs.uk/services/future-gp-it-systems-and-services');
+});
+
+test('should navigate to about GPIT futures page', async (t) => {
+  await pageSetup(t);
+  const getLocation = ClientFunction(() => document.location.href);
+  const cookiesLink = Selector('[data-test-id="legal-panel"] span:nth-child(2) > a');
+  await t
+    .expect(cookiesLink.exists).ok()
+    .click(cookiesLink)
+    .expect(getLocation()).contains('https://digital.nhs.uk/about-nhs-digital/privacy-and-cookies');
+});
+
+test('should show legal banner', async (t) => {
+  await pageSetup(t);
+  const legalText = Selector('[data-test-id="legal-panel"] span:nth-child(1)');
+  await t
+    .expect(legalText.innerText).eql('Legal');
+});
+
+fixture('Header')
+  .afterEach(async (t) => {
+    const isDone = nock.isDone();
+    if (!isDone) {
+      nock.cleanAll();
+    }
+
+    await t.expect(isDone).ok('Not all nock interceptors were used!');
+  });
+
+test('should display BETA banner', async (t) => {
+  await pageSetup(t);
+  const legalText = Selector('[data-test-id="terms-banner"] > div > div > div:nth-child(1)');
+  await t
+    .expect(legalText.innerText).eql('BETA');
+});
+
+test('should display General Terms of Use text', async (t) => {
+  await pageSetup(t);
+  const termsOfUseText = Selector('[data-test-id="terms-banner"] > div > div > div:nth-child(2)');
+  await t
+    .expect(termsOfUseText.innerText).eql('By using this site you are accepting the General Terms \
+      of Use which you can view by downloading this PDF. The Cookies Policy and Privacy Policy can be \
+      accessed using the links at the bottom of the page.');
```

Comment:
Will it not match without these slashes?

Reply:
you can't have line breaks inside string without slash